src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -90,8 +90,11 @@ FUZZ_TARGET(coinselection)
     // Run coinselection algorithms
     const auto result_bnb = SelectCoinsBnB(group_pos, target, cost_of_change, MAX_STANDARD_TX_WEIGHT);
 
-    auto result_srd = SelectCoinsSRD(group_pos, target, fast_random_context, MAX_STANDARD_TX_WEIGHT);
-    if (result_srd) result_srd->ComputeAndSetWaste(cost_of_change, cost_of_change, 0);
+    auto result_srd = SelectCoinsSRD(group_pos, target, coin_params.m_change_fee, fast_random_context, MAX_STANDARD_TX_WEIGHT);
+    if (result_srd) {
+        assert(result_srd->GetChange(CAmount{294}, coin_params.m_change_fee) > 0); // Shows that SRD always selects enough for a non-dust P2WPKH change output
```

Comment:
Why 294?

Reply:
Answering myself. Dust limit for P2WPKH is 294 = (67 vbytes for input + 31 vbytes for output) * 3sat/vb

Reply:
But why use P2WPKH in particular, when `change_output_size` is a random int for 10 to 1000

Reply:
This parameter is the minimum viable change, which for this test, I think makes sense to actually be as high as possible, rather than dependent on the change output itself. The new behavior is that SRD will never make a change output that is smaller than `CHANGE_LOWER`, so I think it would make sense to set this to `CHANGE_LOWER`.

Reply:
@S3RK: Yeah, you got it right.

@achow101: Followed your suggestion and updated the minimum to `CHANGE_LOWER`.